docs/release.adoc

```diff
@@ -6,7 +6,35 @@ IMPORTANT: Guillotine 7 requires XP 7.14.0 or higher.
 
 === Release features
 
-SiteKey:: A new `siteKey` argument has been added to the `guillotine` field. This argument is optional and can be used instead of the `X-Guillotine-SiteKey` header.
+Type definitions on NPM:: Guillotine type definitions are now https://www.npmjs.com/package/@enonic-types/guillotine[available on NPM^]. You can install them using the following command:
+
+  npm install @enonic/guillotine-types --save-dev
```

Comment:
```suggestion
  `npm install @enonic/guillotine-types --save-dev`
```

Reply:
It's not necessary when there's 2 space indentation. It will render as a code block.